Challenge 3

Shuqihong opened the pull request.

Shuqihong committed:
deleted challenge2

Shuqihong committed:
deleted

Shuqihong committed:
Challenge 3 Solution

SidharthSaluja reviewed (commented):
Any reason why you deleted challenge 2?

Shuqihong commented:
I first made a wrong name for challenge 2 which had space in it, but I
pushed it to my github. Then I renamed it without space and pushed the new
file again. But I found that there were two challenge 2 files in my github.
One was with space and one was without it. So I deleted the with space file.

On Wed, Jun 7, 2023 at 10:12 PM Sidharth Saluja ***@***.***>
wrote:

> ***@***.**** commented on this pull request.
>
> Any reason why you deleted challenge 2?
>
> —
> Reply to this email directly, view it on GitHub
> <https://nam10.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FDACSS%2FDACSS_601_Summer2023_Sec1%2Fpull%2F77%23pullrequestreview-1468872796&data=05%7C01%7Cshuqihong%40umass.edu%7Cf4d082c196eb473b238808db67c5bff8%7C7bd08b0b33954dc194bbd0b2e56a497f%7C0%7C0%7C638217871264231513%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=6NfFF76AHDewFSDmKi7JHgwF7qbz5tQqClc%2BvesbAak%3D&reserved=0>,
> or unsubscribe
> <https://nam10.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FBAGQOFRCDSWRFZM2WFVI753XKEYHFANCNFSM6AAAAAAY6SXJFM&data=05%7C01%7Cshuqihong%40umass.edu%7Cf4d082c196eb473b238808db67c5bff8%7C7bd08b0b33954dc194bbd0b2e56a497f%7C0%7C0%7C638217871264231513%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=GkSYyjRlHvkm1whYFyYHNR33QKTluyf3EqOXijgvQO8%3D&reserved=0>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

SidharthSaluja reviewed (approved):
Make sense
